pyproject.toml

```diff
@@ -0,0 +1,204 @@
+#
+# Ralph package
+#
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ralph-malph"
+version = "3.9.0"
+description = "Ralph, the ultimate Learning Record Store (and more!) for your learning analytics."
+readme = "README.md"
+authors = [
+    { "name" = "Open FUN (France Université Numérique)", "email" = "fun.dev@fun-mooc.fr" }
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+requires-python = ">=3.7"
+license = { file = "LICENSE.md" }
+keywords = ["LRS", "Analytics", "xAPI", "Open edX"]
+dependencies = [
+    # By default, we only consider core dependencies required to use Ralph as a
+    # library (mostly models).
+    "langcodes>=3.2.0",
+    "pydantic[dotenv,email]>=1.10.0, <2.0",
+    "rfc3987>=1.3.0",
+]
+
+[project.urls]
+"Homepage" = "https://openfun.github.io/ralph/"
+"Bug Tracker" = "https://github.com/openfun/ralph/issues/new"
```

Comment:
Why not `https://github.com/openfun/ralph/issues` ? When people would click on it, it would allow them first to check if an issue already exists for the bug they want to share about

Reply:
Yes indeed!